nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -96,18 +96,27 @@ define([
             configNode.classList.add('hidden');
             container.appendChild(configNode);
 
-            return Util.getMissingFiles(Array.from(allFiles))
-                .catch((error) => {
-                    // if the missing files call fails, just continue and let the cell render.
-                    console.error('Unable to get missing files from the Staging Service', error);
-                })
-                .then((missingFiles = []) => {
-                    unavailableFiles = new Set(missingFiles);
-                    // Do a validation for all input parameters on all file types
-                    // This is then sent to the fileTypePanel to initialize properly with
-                    // pass or fail for each side, to properly render pass or fail icons.
-                    return Util.evaluateConfigReadyState(model, specs, unavailableFiles);
-                })
+            let startupPromise;
+            if (!viewOnly) {
```

Comment:
As in other places, only do the parameter state check if we're not in view only mode.